### PR TITLE
Fix bessctl gates tracking

### PR DESCRIPTION
**Pull request by ddiproietto**
This should fix Issue #398

### CODE REVIEW

File: bessctl/conf/samples/exactmatch.bess [hunk 1]
```diff
@@ -42,3 +42,9 @@ em.add(fields=[chr(17), aton('192.168.1.123'), aton('12.34.56.78')], gate=3)
 em.delete(fields=[chr(17), aton('192.168.1.123'), aton('12.34.56.78')])
 
 em.set_default_gate(gate=5)
+
+# Test gates tracking.
```
Comment (sangjinhan) [inline bessctl/conf/samples/exactmatch.bess]: Did you intend to keep this block?

Comment (ddiproietto) [inline bessctl/conf/samples/exactmatch.bess]: Yes, that was intentional.  I realize that this has nothing to do with exactmatch, it's probably better to move it.

Do you have any suggestions?

Thanks!

Comment (sangjinhan) [inline bessctl/conf/samples/exactmatch.bess]: Let's not include this block in this sample. Otherwise everything looks great to me :-)

Comment (ddiproietto) [inline bessctl/conf/samples/exactmatch.bess]: Great, I removed it

**Comment by sangjinhan** [comment]
Merged, thanks!
tumblr_m4so38z8pk1qbe5pxo1_1280

**Pull request merged**